modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/GoClientCodegen.java

```diff
@@ -565,13 +565,6 @@ private String constructExampleCode(CodegenModel codegenModel, HashMap<String, C
             processedModelMap.put(model, 1);
         }
 
-        example = "" + goImportAlias + "." + codegenModel.name + "{";
-        List<String> propertyExamples = new ArrayList<>();
-        for (CodegenProperty codegenProperty : codegenModel.allVars) {
-            propertyExamples.add(codegenProperty.name + ": " + constructExampleCode(codegenProperty, modelMaps, processedModelMap));
-        }
```

Comment:
We should probably include at least required arguments.

Reply:
Yeah I wasn't too sure what to do here, keeping simplicity of example, where you'd have to add manually the required arguments, or having everything in a single long line like before, and losing that readability.

I'll try to use the constructor with required arguments to see how it looks